src/main/java/org/agbrothers/englishbot/repository/WordRepository.java

```diff
@@ -2,9 +2,19 @@
 
 import org.agbrothers.englishbot.entity.Word;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
+
 @Repository
 public interface WordRepository extends JpaRepository<Word, Long> {
     Word findByEnglishValue(String englishValue);
+
+    @Query("SELECT id from Word")
+    List<Long> getAllIdWord();
+
+    @Query("from Word w where w.id in :ids")
+    List<Word> findWordsForLesson(@Param("ids") Iterable<Long> ids);
```

Comment:
I suppose, you don't have to write a custom @Query here. Please, take a look https://stackoverflow.com/questions/34837725/spring-data-repositories-find-where-field-in-list